miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/PoolRestController.java

```diff
@@ -317,4 +338,67 @@ public void bulkDelete(@RequestBody(required = true) List<Long> ids) throws IOEx
     poolService.bulkDelete(pools);
   }
 
+  private static Stream<Sample> getSamples(Pool pool) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this method into the anonymous class declared at line 358. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3398)